src/scripts/cfo.zig

```diff
@@ -683,6 +681,34 @@ fn run_fuzzers_prepare_tasks(tasks: *Tasks, shell: *Shell, gh_token: ?[]const u8
     }
 }
 
+/// Garbage-collect checkouts which don't have any running tasks.
+/// Otherwise we reuse checkouts as long as possible, to minimize time spent recompiling fuzzers.
+fn run_fuzzers_cleanup(tasks: *Tasks, shell: *Shell, gh_token: ?[]const u8) !void {
+    if (gh_token) |_| {
+        const working_directory = try shell.cwd.openDir("./working", .{ .iterate = true });
+        var checkouts = working_directory.iterate();
+        while (try checkouts.next()) |checkout| {
+            if (checkout.kind == .directory) {
+                if (std.mem.eql(u8, checkout.name, "main")) continue;
```

Comment:
Can we collect `main`at least every 100th iteration or something? I _think_ `.zig-cache` will grow infinitely over wise, I don't think Zig implements GC of incremental compilation state?

Reply:
That's a great idea! The supervisor script can just randomly delete the entire working directory periodically. That is much simpler than my approach, and it solves the state issue also.